Assets/scss/components/_assignment.scss

```diff
@@ -0,0 +1,189 @@
+.component-wrapper {
+    display: flex;
+    flex-wrap: wrap;
+    font-family: $openSans;
+
+    @include max-screen(lg) {
+        flex-direction: column;
+    }
+
+    .text-area {
+        width: 33%;
+        padding: toRem(80);
+
+        h1 {
+            font-size: toRem(32);
+            font-weight: bold;
+            margin-bottom: toRem(20);
+        }
+
+        p {
+            font-size: toRem(16);
+            line-height: 1.5;
+        }
+
+        @include max-screen(lg) {
+            width: 100%;
+            padding: toRem(20);
+        }
+    }
+
+    .image-area {
+        width: 67%;
+        position: relative;
+        background-image: url('../../images/home_image.png');
```

Comment:
@dkdeepesh  Why you are using background-image instead of img tag, because later on we are implementing the same comp in AEM and it should be an authorable field. if you want to use background-image then store value in a css variable in inline css of that html element. 
Cc:-  @jvaishnavrp  @msharmaRP

Reply:
I have created a new PR for the above solution, incorporating all the required changes as requested.
https://github.com/jvaishnavrp/FETraining/pull/31
Cc: - @mathrik @Lalit-gour1 @zmanning29 @jvaishnavrp